tests/ui/test_validation_ui.py

```diff
@@ -47,6 +54,20 @@ def test_dashboard_validation_ui(self, setup_ui_class):
         else:
             validation_ui_obj.verification_ui()
 
+        if ocs_version >= version.VERSION_4_13:
+            logger.info(
+                "Verify GET requests initiated by kube-probe on odf-console pod"
+            )
+            pod_odf_console_name = get_pod_name_by_pattern("odf-console")
+            pod_odf_console_logs = get_pod_logs(pod_name=pod_odf_console_name[0])
+            if (
+                re.search(
+                    "GET /plugin-manifest.json HTTP.*kube-probe", pod_odf_console_logs
+                )
+                is None
+            ):
+                raise ValueError("GET request initiated by kube-probe does not exist")
+
```

Comment:
Shouldn't this verification be done before validating the dashboard on UI?

Reply:
I agree with @am-agrawa 
@OdedViner Please change this verification before dashboard validation

Reply:
done